[2023-07-08T00:03:03.012+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_scrap_biobio_v11.scrap_data scheduled__2023-07-07T00:00:00+00:00 [queued]>
[2023-07-08T00:03:03.021+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_scrap_biobio_v11.scrap_data scheduled__2023-07-07T00:00:00+00:00 [queued]>
[2023-07-08T00:03:03.022+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-08T00:03:03.038+0000] {taskinstance.py:1327} INFO - Executing <Task(_PythonVirtualenvDecoratedOperator): scrap_data> on 2023-07-07 00:00:00+00:00
[2023-07-08T00:03:03.042+0000] {standard_task_runner.py:57} INFO - Started process 2135 to run task
[2023-07-08T00:03:03.045+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_scrap_biobio_v11', 'scrap_data', 'scheduled__2023-07-07T00:00:00+00:00', '--job-id', '125', '--raw', '--subdir', 'DAGS_FOLDER/dag_scrap_biobio.py', '--cfg-path', '/tmp/tmp57kqfg4k']
[2023-07-08T00:03:03.046+0000] {standard_task_runner.py:85} INFO - Job 125: Subtask scrap_data
[2023-07-08T00:03:03.080+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_scrap_biobio_v11.scrap_data scheduled__2023-07-07T00:00:00+00:00 [running]> on host 3371e7602d27
[2023-07-08T00:03:03.143+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_scrap_biobio_v11' AIRFLOW_CTX_TASK_ID='scrap_data' AIRFLOW_CTX_EXECUTION_DATE='2023-07-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-07-07T00:00:00+00:00'
[2023-07-08T00:03:03.144+0000] {process_utils.py:181} INFO - Executing cmd: /usr/local/bin/python -m virtualenv /tmp/venvxk70fq6b
[2023-07-08T00:03:03.154+0000] {process_utils.py:185} INFO - Output:
[2023-07-08T00:03:03.928+0000] {process_utils.py:189} INFO - created virtual environment CPython3.7.17.final.0-64 in 169ms
[2023-07-08T00:03:03.928+0000] {process_utils.py:189} INFO -   creator CPython3Posix(dest=/tmp/venvxk70fq6b, clear=False, no_vcs_ignore=False, global=False)
[2023-07-08T00:03:03.928+0000] {process_utils.py:189} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2023-07-08T00:03:03.928+0000] {process_utils.py:189} INFO -     added seed packages: pip==23.1, setuptools==67.6.1, wheel==0.40.0
[2023-07-08T00:03:03.928+0000] {process_utils.py:189} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2023-07-08T00:03:03.986+0000] {process_utils.py:181} INFO - Executing cmd: /tmp/venvxk70fq6b/bin/pip install -r /tmp/venvxk70fq6b/requirements.txt
[2023-07-08T00:03:03.996+0000] {process_utils.py:185} INFO - Output:
[2023-07-08T00:03:04.895+0000] {process_utils.py:189} INFO - WARNING: The directory '/home/***/.cache/pip' or its parent directory is not owned or is not writable by the current user. The cache has been disabled. Check the permissions and owner of that directory. If executing pip with sudo, you should use sudo's -H flag.
[2023-07-08T00:03:05.136+0000] {process_utils.py:189} INFO - Collecting selenium (from -r /tmp/venvxk70fq6b/requirements.txt (line 1))
[2023-07-08T00:03:05.268+0000] {process_utils.py:189} INFO -   Downloading selenium-4.10.0-py3-none-any.whl (6.7 MB)
[2023-07-08T00:03:05.470+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 6.7/6.7 MB 33.5 MB/s eta 0:00:00
[2023-07-08T00:03:05.520+0000] {process_utils.py:189} INFO - Collecting dateparser (from -r /tmp/venvxk70fq6b/requirements.txt (line 2))
[2023-07-08T00:03:05.549+0000] {process_utils.py:189} INFO -   Downloading dateparser-1.1.8-py2.py3-none-any.whl (293 kB)
[2023-07-08T00:03:05.557+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 293.8/293.8 kB 53.5 MB/s eta 0:00:00
[2023-07-08T00:03:05.635+0000] {process_utils.py:189} INFO - Collecting urllib3[socks]<3,>=1.26 (from selenium->-r /tmp/venvxk70fq6b/requirements.txt (line 1))
[2023-07-08T00:03:05.659+0000] {process_utils.py:189} INFO -   Downloading urllib3-2.0.3-py3-none-any.whl (123 kB)
[2023-07-08T00:03:05.664+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 123.6/123.6 kB 36.2 MB/s eta 0:00:00
[2023-07-08T00:03:05.708+0000] {process_utils.py:189} INFO - Collecting trio~=0.17 (from selenium->-r /tmp/venvxk70fq6b/requirements.txt (line 1))
[2023-07-08T00:03:05.736+0000] {process_utils.py:189} INFO -   Downloading trio-0.22.1-py3-none-any.whl (399 kB)
[2023-07-08T00:03:05.749+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 399.3/399.3 kB 38.1 MB/s eta 0:00:00
[2023-07-08T00:03:05.782+0000] {process_utils.py:189} INFO - Collecting trio-websocket~=0.9 (from selenium->-r /tmp/venvxk70fq6b/requirements.txt (line 1))
[2023-07-08T00:03:05.810+0000] {process_utils.py:189} INFO -   Downloading trio_websocket-0.10.3-py3-none-any.whl (17 kB)
[2023-07-08T00:03:05.858+0000] {process_utils.py:189} INFO - Collecting certifi>=2021.10.8 (from selenium->-r /tmp/venvxk70fq6b/requirements.txt (line 1))
[2023-07-08T00:03:05.887+0000] {process_utils.py:189} INFO -   Downloading certifi-2023.5.7-py3-none-any.whl (156 kB)
[2023-07-08T00:03:05.896+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 157.0/157.0 kB 22.7 MB/s eta 0:00:00
[2023-07-08T00:03:05.942+0000] {process_utils.py:189} INFO - Collecting python-dateutil (from dateparser->-r /tmp/venvxk70fq6b/requirements.txt (line 2))
[2023-07-08T00:03:05.971+0000] {process_utils.py:189} INFO -   Downloading python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
[2023-07-08T00:03:05.979+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 247.7/247.7 kB 47.0 MB/s eta 0:00:00
[2023-07-08T00:03:06.088+0000] {process_utils.py:189} INFO - Collecting pytz (from dateparser->-r /tmp/venvxk70fq6b/requirements.txt (line 2))
[2023-07-08T00:03:06.118+0000] {process_utils.py:189} INFO -   Downloading pytz-2023.3-py2.py3-none-any.whl (502 kB)
[2023-07-08T00:03:06.130+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 502.3/502.3 kB 51.7 MB/s eta 0:00:00
[2023-07-08T00:03:06.868+0000] {process_utils.py:189} INFO - Collecting regex!=2019.02.19,!=2021.8.27 (from dateparser->-r /tmp/venvxk70fq6b/requirements.txt (line 2))
[2023-07-08T00:03:06.895+0000] {process_utils.py:189} INFO -   Downloading regex-2023.6.3-cp37-cp37m-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (755 kB)
[2023-07-08T00:03:06.913+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 755.7/755.7 kB 48.6 MB/s eta 0:00:00
[2023-07-08T00:03:06.973+0000] {process_utils.py:189} INFO - Collecting tzlocal (from dateparser->-r /tmp/venvxk70fq6b/requirements.txt (line 2))
[2023-07-08T00:03:06.995+0000] {process_utils.py:189} INFO -   Downloading tzlocal-5.0.1-py3-none-any.whl (20 kB)
[2023-07-08T00:03:07.058+0000] {process_utils.py:189} INFO - Collecting attrs>=20.1.0 (from trio~=0.17->selenium->-r /tmp/venvxk70fq6b/requirements.txt (line 1))
[2023-07-08T00:03:07.083+0000] {process_utils.py:189} INFO -   Downloading attrs-23.1.0-py3-none-any.whl (61 kB)
[2023-07-08T00:03:07.087+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.2/61.2 kB 27.4 MB/s eta 0:00:00
[2023-07-08T00:03:07.127+0000] {process_utils.py:189} INFO - Collecting sortedcontainers (from trio~=0.17->selenium->-r /tmp/venvxk70fq6b/requirements.txt (line 1))
[2023-07-08T00:03:07.158+0000] {process_utils.py:189} INFO -   Downloading sortedcontainers-2.4.0-py2.py3-none-any.whl (29 kB)
[2023-07-08T00:03:07.201+0000] {process_utils.py:189} INFO - Collecting idna (from trio~=0.17->selenium->-r /tmp/venvxk70fq6b/requirements.txt (line 1))
[2023-07-08T00:03:07.227+0000] {process_utils.py:189} INFO -   Downloading idna-3.4-py3-none-any.whl (61 kB)
[2023-07-08T00:03:07.231+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.5/61.5 kB 26.3 MB/s eta 0:00:00
[2023-07-08T00:03:07.269+0000] {process_utils.py:189} INFO - Collecting outcome (from trio~=0.17->selenium->-r /tmp/venvxk70fq6b/requirements.txt (line 1))
[2023-07-08T00:03:07.300+0000] {process_utils.py:189} INFO -   Downloading outcome-1.2.0-py2.py3-none-any.whl (9.7 kB)
[2023-07-08T00:03:07.340+0000] {process_utils.py:189} INFO - Collecting sniffio (from trio~=0.17->selenium->-r /tmp/venvxk70fq6b/requirements.txt (line 1))
[2023-07-08T00:03:07.369+0000] {process_utils.py:189} INFO -   Downloading sniffio-1.3.0-py3-none-any.whl (10 kB)
[2023-07-08T00:03:07.434+0000] {process_utils.py:189} INFO - Collecting exceptiongroup>=1.0.0rc9 (from trio~=0.17->selenium->-r /tmp/venvxk70fq6b/requirements.txt (line 1))
[2023-07-08T00:03:07.459+0000] {process_utils.py:189} INFO -   Downloading exceptiongroup-1.1.2-py3-none-any.whl (14 kB)
[2023-07-08T00:03:07.503+0000] {process_utils.py:189} INFO - Collecting wsproto>=0.14 (from trio-websocket~=0.9->selenium->-r /tmp/venvxk70fq6b/requirements.txt (line 1))
[2023-07-08T00:03:07.538+0000] {process_utils.py:189} INFO -   Downloading wsproto-1.2.0-py3-none-any.whl (24 kB)
[2023-07-08T00:03:07.603+0000] {process_utils.py:189} INFO - Collecting pysocks!=1.5.7,<2.0,>=1.5.6 (from urllib3[socks]<3,>=1.26->selenium->-r /tmp/venvxk70fq6b/requirements.txt (line 1))
[2023-07-08T00:03:07.643+0000] {process_utils.py:189} INFO -   Downloading PySocks-1.7.1-py3-none-any.whl (16 kB)
[2023-07-08T00:03:07.691+0000] {process_utils.py:189} INFO - Collecting six>=1.5 (from python-dateutil->dateparser->-r /tmp/venvxk70fq6b/requirements.txt (line 2))
[2023-07-08T00:03:07.715+0000] {process_utils.py:189} INFO -   Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)
[2023-07-08T00:03:07.773+0000] {process_utils.py:189} INFO - Collecting backports.zoneinfo (from tzlocal->dateparser->-r /tmp/venvxk70fq6b/requirements.txt (line 2))
[2023-07-08T00:03:07.803+0000] {process_utils.py:189} INFO -   Downloading backports.zoneinfo-0.2.1-cp37-cp37m-manylinux1_x86_64.whl (70 kB)
[2023-07-08T00:03:07.807+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 70.7/70.7 kB 53.3 MB/s eta 0:00:00
[2023-07-08T00:03:07.927+0000] {process_utils.py:189} INFO - Collecting importlib-metadata (from attrs>=20.1.0->trio~=0.17->selenium->-r /tmp/venvxk70fq6b/requirements.txt (line 1))
[2023-07-08T00:03:07.953+0000] {process_utils.py:189} INFO -   Downloading importlib_metadata-6.7.0-py3-none-any.whl (22 kB)
[2023-07-08T00:03:08.004+0000] {process_utils.py:189} INFO - Collecting h11<1,>=0.9.0 (from wsproto>=0.14->trio-websocket~=0.9->selenium->-r /tmp/venvxk70fq6b/requirements.txt (line 1))
[2023-07-08T00:03:08.034+0000] {process_utils.py:189} INFO -   Downloading h11-0.14.0-py3-none-any.whl (58 kB)
[2023-07-08T00:03:08.036+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 58.3/58.3 kB 188.7 MB/s eta 0:00:00
[2023-07-08T00:03:08.112+0000] {process_utils.py:189} INFO - Collecting typing-extensions (from h11<1,>=0.9.0->wsproto>=0.14->trio-websocket~=0.9->selenium->-r /tmp/venvxk70fq6b/requirements.txt (line 1))
[2023-07-08T00:03:08.137+0000] {process_utils.py:189} INFO -   Downloading typing_extensions-4.7.1-py3-none-any.whl (33 kB)
[2023-07-08T00:03:08.218+0000] {process_utils.py:189} INFO - Collecting zipp>=0.5 (from importlib-metadata->attrs>=20.1.0->trio~=0.17->selenium->-r /tmp/venvxk70fq6b/requirements.txt (line 1))
[2023-07-08T00:03:08.247+0000] {process_utils.py:189} INFO -   Downloading zipp-3.15.0-py3-none-any.whl (6.8 kB)
[2023-07-08T00:03:08.497+0000] {process_utils.py:189} INFO - Installing collected packages: sortedcontainers, pytz, zipp, urllib3, typing-extensions, sniffio, six, regex, pysocks, idna, exceptiongroup, certifi, backports.zoneinfo, tzlocal, python-dateutil, importlib-metadata, h11, wsproto, dateparser, attrs, outcome, trio, trio-websocket, selenium
[2023-07-08T00:03:09.980+0000] {process_utils.py:189} INFO - Successfully installed attrs-23.1.0 backports.zoneinfo-0.2.1 certifi-2023.5.7 dateparser-1.1.8 exceptiongroup-1.1.2 h11-0.14.0 idna-3.4 importlib-metadata-6.7.0 outcome-1.2.0 pysocks-1.7.1 python-dateutil-2.8.2 pytz-2023.3 regex-2023.6.3 selenium-4.10.0 six-1.16.0 sniffio-1.3.0 sortedcontainers-2.4.0 trio-0.22.1 trio-websocket-0.10.3 typing-extensions-4.7.1 tzlocal-5.0.1 urllib3-2.0.3 wsproto-1.2.0 zipp-3.15.0
[2023-07-08T00:03:10.131+0000] {process_utils.py:189} INFO - 
[2023-07-08T00:03:10.131+0000] {process_utils.py:189} INFO - [notice] A new release of pip is available: 23.1 -> 23.1.2
[2023-07-08T00:03:10.131+0000] {process_utils.py:189} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2023-07-08T00:03:10.270+0000] {process_utils.py:181} INFO - Executing cmd: /tmp/venvxk70fq6b/bin/python /tmp/venvxk70fq6b/script.py /tmp/venvxk70fq6b/script.in /tmp/venvxk70fq6b/script.out /tmp/venvxk70fq6b/string_args.txt
[2023-07-08T00:03:10.280+0000] {process_utils.py:185} INFO - Output:
[2023-07-08T00:03:21.028+0000] {process_utils.py:189} INFO - 1) BROWSER OK
[2023-07-08T00:03:21.028+0000] {process_utils.py:189} INFO - 2) GET OK
[2023-07-08T00:03:21.028+0000] {process_utils.py:189} INFO - 3) BUTTON OK
[2023-07-08T00:03:21.028+0000] {process_utils.py:189} INFO - 4) CLICK OK
[2023-07-08T00:03:21.028+0000] {process_utils.py:189} INFO - 5) FETCH OK
[2023-07-08T00:03:21.028+0000] {process_utils.py:189} INFO - 6) FETCH 2 OK
[2023-07-08T00:03:21.028+0000] {process_utils.py:189} INFO - 4) CLICK OK
[2023-07-08T00:03:21.029+0000] {process_utils.py:189} INFO - 5) FETCH OK
[2023-07-08T00:03:21.029+0000] {process_utils.py:189} INFO - 6) FETCH 2 OK
[2023-07-08T00:03:21.029+0000] {process_utils.py:189} INFO - 4) CLICK OK
[2023-07-08T00:03:21.029+0000] {process_utils.py:189} INFO - 5) FETCH OK
[2023-07-08T00:03:21.029+0000] {process_utils.py:189} INFO - 6) FETCH 2 OK
[2023-07-08T00:03:21.029+0000] {process_utils.py:189} INFO - 4) CLICK OK
[2023-07-08T00:03:21.029+0000] {process_utils.py:189} INFO - 5) FETCH OK
[2023-07-08T00:03:21.029+0000] {process_utils.py:189} INFO - 6) FETCH 2 OK
[2023-07-08T00:03:21.029+0000] {process_utils.py:189} INFO - 4) CLICK OK
[2023-07-08T00:03:21.029+0000] {process_utils.py:189} INFO - 5) FETCH OK
[2023-07-08T00:03:21.029+0000] {process_utils.py:189} INFO - 6) FETCH 2 OK
[2023-07-08T00:03:21.029+0000] {process_utils.py:189} INFO - Loading data...
[2023-07-08T00:03:21.030+0000] {process_utils.py:189} INFO - SCRAPPED: Tras el ciberataque que afectó a sus servicios digitales, el Instituto de Desarrollo Agropecuario (Indap), afirmó que se encuentran en la fase final de la recuperación de sus sistemas informáticos. Nacional A 19 días desde que el Instituto de Desarrollo Agropecuario (Indap) sufriera un incidente de ciberataque que afectó el funcionamiento de su sitio web, la entidad comunicó que la totalidad de sus servicios digitales estarán en condiciones de funcionar para la próxima semana. A su vez, este viernes informaron que, junto con el restablecimiento de su página web, el servicio MI INDAP ya está disponible para los usuarios. Además, afirmaron que el 100% de los vencimientos de los créditos Indap se prorrogarán hasta el 10 de agosto, y en el caso de los afectados por las inundaciones hasta el 10 de octubre del presente año. Asimismo, ningún usuario incurrirá en mora ni cambiará su categoría de riesgo crediticio debido al incidente. La senadora de Renovación Nacional, que presidió la Comisión de Agricultura el año pasado, Carmen Gloria Aravena, afirmó que este bloqueo ha afectado a muchas familias que se dedican a la agricultura, ya que no han podido recurrir a los créditos Indap. El presidente de la Comisión de Agricultura, el diputado UDI, Juan Antonio Coloma, informó que el comité ha estado sesionado en las regiones de O’Higgins y del Maule, constatando en terreno lo mucho que ha afectado a los usuarios de Indap no poder recurrir a créditos ni seguros ante el sistema frontal. Por su parte, el diputado independiente de la bancada de Convergencia Social, Patricio Rosas, afirmó que esto demuestra la evolución de la criminalidad y que el Estado debe mejorar su ciberseguridad. Por tratarse de un evento asociado a ciberseguridad, este incidente está siendo investigado por el Ministerio Público y por el Equipo de Respuesta ante Incidentes de Seguridad Informática del Ministerio del Interior y Seguridad Pública.
[2023-07-08T00:03:21.030+0000] {process_utils.py:189} INFO - Indap anuncia fecha en que sus servicios digitales volverán a funcionar completamente tras ciberataque https://www.biobiochile.cl/noticias/nacional/chile/2023/07/07/indap-anuncia-fecha-en-que-sus-servicios-digitales-volveran-a-funcionar-completamente-tras-ciberataque.shtml
[2023-07-08T00:03:21.031+0000] {process_utils.py:189} INFO - Traceback (most recent call last):
[2023-07-08T00:03:21.031+0000] {process_utils.py:189} INFO -   File "/tmp/venvxk70fq6b/script.py", line 160, in <module>
[2023-07-08T00:03:21.031+0000] {process_utils.py:189} INFO -     res = scrapData(*arg_dict["args"], **arg_dict["kwargs"])
[2023-07-08T00:03:21.031+0000] {process_utils.py:189} INFO -   File "/tmp/venvxk70fq6b/script.py", line 138, in scrapData
[2023-07-08T00:03:21.031+0000] {process_utils.py:189} INFO -     title = str(elem.find_element(By.CLASS_NAME, 'article-title').text)
[2023-07-08T00:03:21.031+0000] {process_utils.py:189} INFO -   File "/tmp/venvxk70fq6b/lib/python3.7/site-packages/selenium/webdriver/remote/webelement.py", line 417, in find_element
[2023-07-08T00:03:21.031+0000] {process_utils.py:189} INFO -     return self._execute(Command.FIND_CHILD_ELEMENT, {"using": by, "value": value})["value"]
[2023-07-08T00:03:21.031+0000] {process_utils.py:189} INFO -   File "/tmp/venvxk70fq6b/lib/python3.7/site-packages/selenium/webdriver/remote/webelement.py", line 395, in _execute
[2023-07-08T00:03:21.031+0000] {process_utils.py:189} INFO -     return self._parent.execute(command, params)
[2023-07-08T00:03:21.031+0000] {process_utils.py:189} INFO -   File "/tmp/venvxk70fq6b/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 346, in execute
[2023-07-08T00:03:21.031+0000] {process_utils.py:189} INFO -     self.error_handler.check_response(response)
[2023-07-08T00:03:21.032+0000] {process_utils.py:189} INFO -   File "/tmp/venvxk70fq6b/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 245, in check_response
[2023-07-08T00:03:21.032+0000] {process_utils.py:189} INFO -     raise exception_class(message, screen, stacktrace)
[2023-07-08T00:03:21.032+0000] {process_utils.py:189} INFO - selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found
[2023-07-08T00:03:21.032+0000] {process_utils.py:189} INFO -   (Session info: headless chrome=114.0.5735.106); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
[2023-07-08T00:03:21.032+0000] {process_utils.py:189} INFO - Stacktrace:
[2023-07-08T00:03:21.032+0000] {process_utils.py:189} INFO - #0 0x56482efc7743 <unknown>
[2023-07-08T00:03:21.032+0000] {process_utils.py:189} INFO - #1 0x56482ecf76ac <unknown>
[2023-07-08T00:03:21.032+0000] {process_utils.py:189} INFO - #2 0x56482ed0467c <unknown>
[2023-07-08T00:03:21.032+0000] {process_utils.py:189} INFO - #3 0x56482ecfc074 <unknown>
[2023-07-08T00:03:21.032+0000] {process_utils.py:189} INFO - #4 0x56482ecfb272 <unknown>
[2023-07-08T00:03:21.032+0000] {process_utils.py:189} INFO - #5 0x56482ecfd235 <unknown>
[2023-07-08T00:03:21.032+0000] {process_utils.py:189} INFO - #6 0x56482ecfd314 <unknown>
[2023-07-08T00:03:21.033+0000] {process_utils.py:189} INFO - #7 0x56482ed31e1b <unknown>
[2023-07-08T00:03:21.033+0000] {process_utils.py:189} INFO - #8 0x56482ed325f1 <unknown>
[2023-07-08T00:03:21.033+0000] {process_utils.py:189} INFO - #9 0x56482ed28eb7 <unknown>
[2023-07-08T00:03:21.033+0000] {process_utils.py:189} INFO - #10 0x56482ed538ed <unknown>
[2023-07-08T00:03:21.033+0000] {process_utils.py:189} INFO - #11 0x56482ed28dca <unknown>
[2023-07-08T00:03:21.033+0000] {process_utils.py:189} INFO - #12 0x56482ed53b4e <unknown>
[2023-07-08T00:03:21.033+0000] {process_utils.py:189} INFO - #13 0x56482ed6aa21 <unknown>
[2023-07-08T00:03:21.033+0000] {process_utils.py:189} INFO - #14 0x56482ed53683 <unknown>
[2023-07-08T00:03:21.033+0000] {process_utils.py:189} INFO - #15 0x56482ed26f63 <unknown>
[2023-07-08T00:03:21.033+0000] {process_utils.py:189} INFO - #16 0x56482ed284d2 <unknown>
[2023-07-08T00:03:21.033+0000] {process_utils.py:189} INFO - #17 0x56482ef95ce8 <unknown>
[2023-07-08T00:03:21.033+0000] {process_utils.py:189} INFO - #18 0x56482ef99a13 <unknown>
[2023-07-08T00:03:21.033+0000] {process_utils.py:189} INFO - #19 0x56482ef994b6 <unknown>
[2023-07-08T00:03:21.034+0000] {process_utils.py:189} INFO - #20 0x56482ef99ef5 <unknown>
[2023-07-08T00:03:21.034+0000] {process_utils.py:189} INFO - #21 0x56482efa06e6 <unknown>
[2023-07-08T00:03:21.034+0000] {process_utils.py:189} INFO - #22 0x56482ef9a273 <unknown>
[2023-07-08T00:03:21.034+0000] {process_utils.py:189} INFO - #23 0x56482ef6f9b6 <unknown>
[2023-07-08T00:03:21.034+0000] {process_utils.py:189} INFO - #24 0x56482efb3be5 <unknown>
[2023-07-08T00:03:21.034+0000] {process_utils.py:189} INFO - #25 0x56482efb3d8e <unknown>
[2023-07-08T00:03:21.034+0000] {process_utils.py:189} INFO - #26 0x56482efc1aef <unknown>
[2023-07-08T00:03:21.034+0000] {process_utils.py:189} INFO - #27 0x7f28177befd4 <unknown>
[2023-07-08T00:03:21.034+0000] {process_utils.py:189} INFO - 
[2023-07-08T00:03:21.109+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/decorators/base.py", line 220, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 374, in execute
    return super().execute(context=serializable_context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 578, in execute_callable
    result = self._execute_python_callable_in_subprocess(python_path, tmp_path)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 434, in _execute_python_callable_in_subprocess
    os.fspath(string_args_path),
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/process_utils.py", line 170, in execute_in_subprocess
    execute_in_subprocess_with_kwargs(cmd, cwd=cwd)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/process_utils.py", line 193, in execute_in_subprocess_with_kwargs
    raise subprocess.CalledProcessError(exit_code, cmd)
subprocess.CalledProcessError: Command '['/tmp/venvxk70fq6b/bin/python', '/tmp/venvxk70fq6b/script.py', '/tmp/venvxk70fq6b/script.in', '/tmp/venvxk70fq6b/script.out', '/tmp/venvxk70fq6b/string_args.txt']' returned non-zero exit status 1.
[2023-07-08T00:03:21.116+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=dag_scrap_biobio_v11, task_id=scrap_data, execution_date=20230707T000000, start_date=20230708T000303, end_date=20230708T000321
[2023-07-08T00:03:21.135+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 125 for task scrap_data (Command '['/tmp/venvxk70fq6b/bin/python', '/tmp/venvxk70fq6b/script.py', '/tmp/venvxk70fq6b/script.in', '/tmp/venvxk70fq6b/script.out', '/tmp/venvxk70fq6b/string_args.txt']' returned non-zero exit status 1.; 2135)
[2023-07-08T00:03:21.158+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-07-08T00:03:21.173+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
